changelogs/unreleased/5651-ignore-plugins-for-modules-that-arent-loaded.yml

```diff
@@ -0,0 +1,6 @@
+---
+description: Ignore plugins for modules that aren't loaded.
+issue-nr: 5651
+issue-repo: inmanta-core
+change-type: patch
+destination-branches: [master, iso6]
```

Comment:
I also wasn't entirely sure. That's why I added Sander as a second reviewer. To me this issue is a blocker for using v2 modules to its full extend and since v2 modules are part of iso5, it should also be merged there. But we can just as well decided not to merge this into iso5 and say that users should upgrade to iso6 get access to this functionality.

Reply:
I would leave it out, partially because it changes a critical component and I'm not sure if we should do that in iso5, partially because of what Bart said on Slack earlier today.

Reply:
I'd also delay merging until we've branched off for the release. I wouldn't be surprised if this causes some fallout.